frontend/src/components/MenuItem.tsx

```diff
@@ -16,12 +16,12 @@ export default function MenuItem({ name, price, picture, addItem }: MenuItemProp
           <Image src={picture} width={imageSize} height={imageSize} alt={name}/>
         </span>
         <span className="menu-pricing-group">
-          <span>Price:</span>
-          <span className="menu-item-price">${price}</span>
+          <span><b>Price:</b></span>
```

Comment:
So unlike semantic tags (span, header, footer, section) I'm not a big fan of tags that affect the style of the content. We should use css to make the headers bold.

Reply:
Copy that. Change is in place now.